starting worker pid=16688 on localhost:11971 at 16:43:18.198
starting worker pid=17108 on localhost:11971 at 16:43:18.216
starting worker pid=3104 on localhost:11971 at 16:43:18.226
starting worker pid=9428 on localhost:11971 at 16:43:18.244
starting worker pid=17304 on localhost:11971 at 16:43:18.249
starting worker pid=12964 on localhost:11971 at 16:43:18.267
starting worker pid=15200 on localhost:11971 at 16:43:18.275
starting worker pid=18032 on localhost:11971 at 16:45:30.202
starting worker pid=11328 on localhost:11971 at 16:45:30.219
starting worker pid=17616 on localhost:11971 at 16:45:30.240
starting worker pid=11048 on localhost:11971 at 16:45:30.241
starting worker pid=3880 on localhost:11971 at 16:45:30.244
starting worker pid=18096 on localhost:11971 at 16:45:30.266
starting worker pid=17944 on localhost:11971 at 16:45:30.273
starting worker pid=16480 on localhost:11971 at 16:45:45.848
starting worker pid=6684 on localhost:11971 at 16:45:45.857
starting worker pid=15296 on localhost:11971 at 16:45:45.868
starting worker pid=17300 on localhost:11971 at 16:45:45.882
starting worker pid=3756 on localhost:11971 at 16:45:45.899
starting worker pid=3544 on localhost:11971 at 16:45:45.907
starting worker pid=10228 on localhost:11971 at 16:45:45.913
starting worker pid=16628 on localhost:11971 at 16:45:55.566
starting worker pid=18072 on localhost:11971 at 16:45:55.566
starting worker pid=18332 on localhost:11971 at 16:45:55.572
starting worker pid=3372 on localhost:11971 at 16:45:55.591
starting worker pid=18376 on localhost:11971 at 16:45:55.601
starting worker pid=13876 on localhost:11971 at 16:45:55.613
starting worker pid=2016 on localhost:11971 at 16:45:55.629
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=11020 on localhost:11971 at 17:50:08.716
starting worker pid=9152 on localhost:11971 at 17:50:08.727
starting worker pid=14532 on localhost:11971 at 17:50:08.745
starting worker pid=10556 on localhost:11971 at 17:50:08.753
starting worker pid=16828 on localhost:11971 at 17:50:08.765
starting worker pid=17056 on localhost:11971 at 17:50:08.777
starting worker pid=17688 on localhost:11971 at 17:50:08.798
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Execution halted
Execution halted
Execution halted
Execution halted
Execution halted
starting worker pid=6260 on localhost:11971 at 22:04:27.424
starting worker pid=18640 on localhost:11971 at 22:04:27.446
starting worker pid=4404 on localhost:11971 at 22:04:27.448
starting worker pid=10628 on localhost:11971 at 22:04:27.479
starting worker pid=1852 on localhost:11971 at 22:04:27.487
starting worker pid=16040 on localhost:11971 at 22:04:27.493
starting worker pid=17016 on localhost:11971 at 22:04:27.513
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=4620 on localhost:11971 at 22:48:59.322
starting worker pid=7188 on localhost:11971 at 22:48:59.323
starting worker pid=15720 on localhost:11971 at 22:48:59.334
starting worker pid=17336 on localhost:11971 at 22:48:59.356
starting worker pid=12916 on localhost:11971 at 22:48:59.366
starting worker pid=19876 on localhost:11971 at 22:48:59.372
starting worker pid=10340 on localhost:11971 at 22:48:59.386
starting worker pid=19464 on localhost:11971 at 22:49:45.933
starting worker pid=16776 on localhost:11971 at 22:49:45.936
starting worker pid=1844 on localhost:11971 at 22:49:45.962
starting worker pid=17680 on localhost:11971 at 22:49:45.973
starting worker pid=12156 on localhost:11971 at 22:49:45.986
starting worker pid=5276 on localhost:11971 at 22:49:45.999
starting worker pid=17796 on localhost:11971 at 22:49:46.008
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
ze(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=16172 on localhost:11971 at 22:50:25.604
starting worker pid=19080 on localhost:11971 at 22:50:25.613
starting worker pid=18340 on localhost:11971 at 22:50:25.625
starting worker pid=18600 on localhost:11971 at 22:50:25.639
starting worker pid=18440 on localhost:11971 at 22:50:25.665
starting worker pid=17616 on localhost:11971 at 22:50:25.666
starting worker pid=20360 on localhost:11971 at 22:50:25.670
starting worker pid=14828 on localhost:11971 at 22:50:48.441
starting worker pid=16936 on localhost:11971 at 22:50:48.445
starting worker pid=19176 on localhost:11971 at 22:50:48.446
starting worker pid=19484 on localhost:11971 at 22:50:48.450
starting worker pid=18740 on localhost:11971 at 22:50:48.459
starting worker pid=11920 on localhost:11971 at 22:50:48.477
starting worker pid=17812 on localhost:11971 at 22:50:48.488
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=16172 on localhost:11971 at 22:54:00.326
starting worker pid=11640 on localhost:11971 at 22:54:00.335
starting worker pid=16988 on localhost:11971 at 22:54:00.349
starting worker pid=8352 on localhost:11971 at 22:54:00.360
starting worker pid=16964 on localhost:11971 at 22:54:00.375
starting worker pid=14528 on localhost:11971 at 22:54:00.387
starting worker pid=19948 on localhost:11971 at 22:54:00.396
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=18396 on localhost:11971 at 22:56:27.361
starting worker pid=9300 on localhost:11971 at 22:56:27.361
starting worker pid=20056 on localhost:11971 at 22:56:27.378
starting worker pid=11364 on localhost:11971 at 22:56:27.381
starting worker pid=16596 on localhost:11971 at 22:56:27.395
starting worker pid=16832 on localhost:11971 at 22:56:27.408
starting worker pid=3820 on localhost:11971 at 22:56:27.417
starting worker pid=5724 on localhost:11971 at 22:57:08.643
starting worker pid=11448 on localhost:11971 at 22:57:08.653
starting worker pid=14420 on localhost:11971 at 22:57:08.670
starting worker pid=8384 on localhost:11971 at 22:57:08.689
starting worker pid=15880 on localhost:11971 at 22:57:08.699
starting worker pid=19484 on localhost:11971 at 22:57:08.721
starting worker pid=16756 on localhost:11971 at 22:57:08.722
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
starting worker pid=17480 on localhost:11971 at 22:59:37.363
starting worker pid=17460 on localhost:11971 at 22:59:37.369
starting worker pid=17592 on localhost:11971 at 22:59:37.383
starting worker pid=16984 on localhost:11971 at 22:59:37.397
starting worker pid=15600 on localhost:11971 at 22:59:37.407
starting worker pid=15416 on localhost:11971 at 22:59:37.422
starting worker pid=16656 on localhost:11971 at 22:59:37.429
starting worker pid=19476 on localhost:11971 at 23:11:28.410
starting worker pid=3552 on localhost:11971 at 23:11:28.415
starting worker pid=3636 on localhost:11971 at 23:11:28.430
starting worker pid=15788 on localhost:11971 at 23:11:28.449
starting worker pid=16180 on localhost:11971 at 23:11:28.465
starting worker pid=8508 on localhost:11971 at 23:11:28.472
starting worker pid=15304 on localhost:11971 at 23:11:28.485
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
ze(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
ze(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
In addition: Warning messages:
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
1: In addition: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
 ning messages:
 localhost:11971 cannot be opened
1: 2: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
 socketConnection(master, port = port, blocking = TRUE, open = "a+b",  : localhost:11971 cannot be opened
 localhost:11971 cannot be opened
3: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
4: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
5: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
6: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
7: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
8: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
9: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
Execution halted
2: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
3: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
4: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
5: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
6: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
7: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
8: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
9: In socketConnection(master, port = port, blocking = TRUE, open = "a+b",  :
  localhost:11971 cannot be opened
Execution halted
